src/assets/scss/components/15-years/15-years.scss

```diff
@@ -59,25 +70,29 @@ $font-family-monospace: 'Roboto Mono', monospace;
 }
 
 @mixin body-text() {
-	font-size: rem-calc(14);
+	font-size: calc(18px / 320 * 100vw);
 	letter-spacing: -0.02em;
 	line-height: 1.35;
 
-	@include breakpoint('large') {
-		font-size: rem-calc(16);
+	// @include breakpoint('large') {
+	// 	font-size: rem-calc(16);
+	// }
+
+	@include breakpoint(small) {
+		font-size: calc(18px / 320 * 100vw);
```

Comment:
Browsers seem to be ignoring these calced values because of the vw unit.

Reply:
Yeah, stylelint is calling these out as invalid calc values because of the mixed px and vw units:
```
function-calc-no-invalid: Expected to be compatible with the left and right argument types of "*" operation.
```

Reply:
I think something like `calc(18 / 320 * 100vw)` (dropping the px) might work?

Reply:
Yeah, dropping px should work. Would it be more appropriate to move these rules into the Header component? Was just speaking w/ the other dev about this